spring-cloud-sleuth-otel/pom.xml

```diff
@@ -54,6 +54,10 @@
 			<groupId>io.opentelemetry</groupId>
 			<artifactId>opentelemetry-api</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>io.opentelemetry</groupId>
+			<artifactId>opentelemetry-api-metrics</artifactId>
```

Comment:
Do we really need this?

Reply:
`Instrumenter` class uses classes from Metrics API. But I will investigate why this explicit dependency was needed.

Reply:
Thanks, it is a little weird that we need this.